### DISCUSSION TITLE
Component proposal: Timeline

### DISCUSSION THREAD

**Discussion by DCruz-FS** [uswds/uswds#5880] [Proposals]
### Component name

Timeline

### Component description

A timeline displays the dates of key events with visuals within a particular historical period in chronological order.

### Why USWDS needs this component

The closest components that USWDS currently has are the Process List component and the Collection component. However, the Process List displays numbers: 1, 2, 3... and are joined with the visual line but not dates. So, maybe we can combine the functionalities of the Process List, Collection and Accordion (that Passed WCAG 2.1 AA) components to create the Timeline Component.

### How the component works

This should clearly convey to users that they are viewing a chronological timeline.

### Real-world examples

I have not encountered a real timeline in the .gov space. I've only seen headings and paragraphs

### When to use this component and when to consider something else

To display a timeline.

### Usability considerations

Instead of using a table, process list or headings and paragraphs.

### Accessibility considerations

Same as accordion.

**Comment by sknep** [comment]
Is what you propose something like: https://www.astrouxds.com/components/timeline/ ?

**Reply by DCruz-FS** [reply]
Good find! This is a little different from what I had in mind. But I like it. It is very interactive. I can see this track version being utilized.
